manila/share/drivers/netapp/dataontap/cluster_mode/lib_multi_svm.py

```diff
@@ -1455,9 +1447,16 @@ def _migration_start_using_svm_migrate(
 
         # 3. Send the migration request to ONTAP.
         try:
+            dest_host = (
+                dest_share_server.get("volume_placement")
+                or dest_share_server["host"]
```

Comment:
what are the conditions that would lead to taking this fallback? I.e. can `volume_placement` ever be empty?